gradle.properties

```diff
@@ -59,3 +59,6 @@ android.enableBuildConfigAsBytecode=true
 
 # By default, the plugin applies itself to all subprojects, but we don't want that as it would cause issues with builds using local AARs
 dependency.analysis.autoapply=false
+
+# Disabled new R8 shrinking for local dependencies as it causes issues with release builds
+android.disableMinifyLocalDependenciesForLibraries=false
```

Comment:
Weird that the value is `false`, it would be more logical to have `true`, but https://developer.android.com/build/releases/gradle-plugin#library-classes-shrunk is not clear about it.

If it fixes the issue, that's fine!